docs/README.rst

```diff
@@ -0,0 +1,82 @@
+.. _readme:
```

Comment:
The formatting needs to be updated copying from https://raw.githubusercontent.com/saltstack-formulas/template-formula/master/docs/README.rst.